hasktorch/src/Torch/TensorFactories.hs

```diff
@@ -54,14 +54,15 @@ linspace start end steps opts = unsafePerformIO $ (cast4 LibTorch.linspace_sslo)
 logspace :: (Scalar a, Scalar b) => a -> b -> Int -> Double -> TensorOptions -> Tensor
 logspace start end steps base opts = unsafePerformIO $ (cast5 LibTorch.logspace_ssldo) start end steps base opts
 
-empty :: [Int] -> TensorOptions -> Tensor
-empty = mkFactoryUnsafe LibTorch.empty_lo
+-- https://github.com/hasktorch/ffi-experimental/pull/57#discussion_r301062033
+-- empty :: [Int] -> TensorOptions -> Tensor
+-- empty = mkFactoryUnsafe LibTorch.empty_lo
 
 eye :: Int -> TensorOptions -> Tensor
 eye dim opts = unsafePerformIO $ (cast2 LibTorch.eye_lo) dim opts
 
-eyerc :: Int -> Int -> TensorOptions -> Tensor
-eyerc rows cols opts = unsafePerformIO $ (cast3 LibTorch.eye_llo) rows cols opts
+eye2 :: Int -> Int -> TensorOptions -> Tensor
+eye2 nrows ncols opts = unsafePerformIO $ (cast3 LibTorch.eye_llo) nrows ncols opts
```

Comment:
How about `eyeSquare` and `eye` for the rectangular version? That's at least more descriptive than `eye2`